make/test.mk

```diff
@@ -80,9 +86,15 @@ olm-integration-setup: olm-integration-cleanup
 
 .PHONY: olm-integration-cleanup
 olm-integration-cleanup: get-test-namespace
-	$(Q)oc login -u system:admin
+	@oc login -u $(OC_LOGIN_USERNAME) -p $(OC_LOGIN_PASSWORD)
+ifeq ($(OPENSHIFT_VERSION),3)
 	$(Q)-oc delete subscription my-devconsole -n operators
 	$(Q)-oc delete catalogsource my-catalog -n olm
+endif
+ifeq ($(OPENSHIFT_VERSION),4)
+	$(Q)-oc delete subscription my-devconsole -n openshift-operators
```

Comment:
just wondering with which user we'll be logged in on openshift 4.x?

Reply:
As for e2e tests, our openshift ci infra gives us a `system:admin`in kubeconfig.

Reply:
@sbose78, don't we need to export it as `KUBECONFIG` env? or it's already exported and logged in as admin user

Reply:
On the openshift CI, it is already done when you specify it is an e2e test.